internal/stores/objects.go

```diff
@@ -321,90 +332,169 @@ func (s *SQLStore) get(key string) (dbObject, error) {
 	return obj, nil
 }
 
-// SlabsForMigration returns up to n slabs which require repair. Only slabs are
```

Comment:
on :316 of this file in `func (s *SQLStore) get(key string) (dbObject, error)` we preload the following `Preload("Slabs.Slab.Shards.DBSector.Contracts.Host")` - I think we should probably stop preloading `Contracts` and `Hosts` 

I have not done this for the time being to not let it go by unnoticed but I think we probably should if we can.